### DISCUSSION TITLE
Welcome to vemto-issues Discussions!

### DISCUSSION THREAD

**Discussion by TiagoSilvaPereira** [TiagoSilvaPereira/vemto-issues#133] [General]
## 👋 Welcome!

  We’re using Discussions as a place to connect with other members of our community. We hope that you:

  * Make suggestions or discuss enhancements
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by asharekh** [comment]
hi, 
i'm using trial, and generation keep give me error. 

when i type composer --v it gives me version

setting for composer is empty in vemto

but still not able to generate the code.

any idea

**Reply by TiagoSilvaPereira** [reply]
Hi @asharekh please try following these steps to solve your issue: https://vemto.app/docs/0.x/configuration
